src/hotspot/os/windows/globals_windows.hpp

```diff
@@ -39,6 +39,10 @@
 product(bool, UseAllWindowsProcessorGroups, false,                        \
         "Use all processor groups on supported Windows versions")         \
                                                                           \
+product(bool, EnableAllLargePageSizes, false, EXPERIMENTAL,               \
+        "Enable support for multiple large page sizes on "                \
+        "Client 11 + and Server 22 + only")                               \
```

Comment:
I don't think "Client 11" and "Server 22" will be clear enough for customers. Consider something like "Windows 11/Windows Server 2022 and later"

Reply:
done